src/gl/gl3device.cpp

```diff
@@ -30,6 +31,80 @@ bool32 needToReadBackTextures;
 int32   alphaFunc;
 float32 alphaRef;
 
+// Helper function to log OpenGL info using SDL logging and optionally to file
+static void
+logGLInfo(void)
+{
+	const char *version     = (const char *)glGetString(GL_VERSION);
+	const char *vendor      = (const char *)glGetString(GL_VENDOR);
+	const char *renderer    = (const char *)glGetString(GL_RENDERER);
+	const char *extensions  = (const char *)glGetString(GL_EXTENSIONS);
+	const char *shadingLang = (const char *)glGetString(GL_SHADING_LANGUAGE_VERSION);
+
+	GLint maxVertexAttribs, maxVertexUniforms, maxFragmentUniforms, maxTextureSize;
+	glGetIntegerv(GL_MAX_VERTEX_ATTRIBS, &maxVertexAttribs);
+	glGetIntegerv(GL_MAX_VERTEX_UNIFORM_VECTORS, &maxVertexUniforms);
+	glGetIntegerv(GL_MAX_FRAGMENT_UNIFORM_VECTORS, &maxFragmentUniforms);
+	glGetIntegerv(GL_MAX_TEXTURE_SIZE, &maxTextureSize);
+
+	// Always log to SDL_Log (shows in logcat on Android)
+	SDL_Log("== OpenGL Info ==");
```

Comment:
librw also supports glfw, so some of this code needs to be surrounded with `#if defined(LIBRW_SDL2) || defined(LIBRW_SDL3)`.